### DISCUSSION TITLE
Naming of Filter Func<T, bool> properties convention is confusing

### DISCUSSION THREAD

**Discussion by buvinghausen** [open-telemetry/opentelemetry-dotnet#3878] [General]
Honestly the name feels like the property belongs in the movie Tenet's inverted world.

Rationale:
If the filter returns true it should *NOT* record the event because it has been filtered out.  That is exactly the opposite of what actually happens.

Rather than invert everyone's code I think you should name the properties something like StartActivity or something where it's implied that true = Record Activity and false = Skip Activity

Language is important and I shouldn't have to read the documentation or code to understand the desired outcomes.

**Comment by Kielek** [comment]
@buvinghausen, could you please check #3816 and provide some feedback?

**Reply by buvinghausen** [reply]
@Kielek done!
